handlers/infobip/infobip.go

```diff
@@ -218,13 +208,18 @@ func (h *handler) SendMsg(ctx context.Context, msg courier.Msg) (courier.MsgStat
 	}
 
 	// build our request
+	var authorization = fmt.Sprintf("App %s", msg.Channel().StringConfigForKey(courier.ConfigAPIKey, ""))
```

Comment:
Let's make the `authorization` an internal variable, not a global one. So it would be like:
```
authorization := fmt.Sprintf("App %s", msg.Channel().StringConfigForKey(courier.ConfigAPIKey, ""))
```